src/browser/components/MattermostView.jsx

```diff
@@ -230,6 +239,22 @@ const MattermostView = createReactClass({
     );
   },
 
+  extractFileURL(message) {
```

Comment:
I feel new functions should be normal functions rather than instance methods. We can test them apart from `<Mattermost>` instance.